changelog.md

```diff
@@ -108,6 +108,20 @@
   users from the use of weak and insecure ciphers while still provides
   adequate compatiblity with the majority of the Internet.
 
+- `net.newContext` drops support for setting the _exact_ version of SSL/TLS.
```

Comment:
Setting only a minimum version is good but perhaps not flexible enough. As it happened in the past, a newer version can have a critical vulnerability that is not present in previous versions. Users might want to either disable that version or explicitly set a list of enabled versions.

Reply:
This is a breaking change, so I want to push it first.

That said, I'm not sure how to add a "max version" parameter, as that would mean we are artificially limiting the maximum version with whatever the stdlib have at the time (assuming a default parameter of type `SslProtVersion`). It might be better to provide "maximum version" as a separate `proc`, so it would be opt-in.

Reply:
I'd also like to be able to set the maximum version, turning off tls1.3 is the only way i know to force a client to only use the cipher you specify, e.g. for tls-psk.